homeassistant/components/sensor/zwave.py

```diff
@@ -18,7 +18,7 @@
 def get_device(node, values, **kwargs):
     """Create zwave entity device."""
     # Generic Device mappings
-    if values.primary.command_class == zwave.const.COMMAND_CLASS_BATTERY:
+    if node.has_command_class(zwave.const.COMMAND_CLASS_BATTERY):
```

Comment:
This isn't equivalent. The original code creates a sensor class if the primary value is a `COMMAND_CLASS_BATTERY` value. This change makes it so that if the node contains `COMMAND_CLASS_BATTERY` at all, then any device for that node will be a ZWaveSensor.